gno.land/pkg/gnoclient/client_txs.go

```diff
@@ -9,17 +10,50 @@ import (
 	"github.com/gnolang/gno/tm2/pkg/std"
 )
 
-// CallCfg contains configuration options for executing a contract call.
-type CallCfg struct {
-	PkgPath        string   // Package path
-	FuncName       string   // Function name
-	Args           []string // Function arguments
-	GasFee         string   // Gas fee
-	GasWanted      int64    // Gas wanted
-	Send           string   // Send amount
-	AccountNumber  uint64   // Account number
-	SequenceNumber uint64   // Sequence number
-	Memo           string   // Memo
+var (
+	errInvalidPkgPath   = errors.New("invalid pkgpath")
+	errInvalidFuncName  = errors.New("invalid function name")
+	errInvalidGasWanted = errors.New("invalid gas wanted")
+	errInvalidGasFee    = errors.New("invalid gas fee")
+	errMissingSigner    = errors.New("missing Signer")
+	errMissingRPCClient = errors.New("missing RPCClient")
+)
```

Comment:
```suggestion
var (
	ErrInvalidPkgPath   = errors.New("invalid pkgpath")
	ErrInvalidFuncName  = errors.New("invalid function name")
	ErrInvalidGasWanted = errors.New("invalid gas wanted")
	ErrInvalidGasFee    = errors.New("invalid gas fee")
	ErrMissingSigner    = errors.New("missing Signer")
	ErrMissingRPCClient = errors.New("missing RPCClient")
)
```

More information about the error would be nice (example: invalid function name: "function_name"), but it is a good start.

Reply:
Since these specific errors just check if the func & path names are empty, I've renamed them: [def2e0b](https://github.com/gnolang/gno/pull/1565/commits/def2e0b206ed84cb9ed1aafb9bacf533fcccc393)

For all real "invalid" function names, like ones starting with numbers, the user will get an error from the VM.